src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -2785,14 +2785,18 @@ namespace NServiceBus.Transports
     {
         protected TransportDefinition() { }
         public abstract string ExampleConnectionStringForErrorMessage { get; }
-        [System.ObsoleteAttribute("Please use `GetOutboundRoutingPolicy` instead. Will be removed in version 7.0.0.", true)]
+        [System.ObsoleteAttribute("Use TransportInfrastructure.OutboundRoutingPolicy.Publishes == OutboundRoutingTyp" +
+            "e.Multicast instead. Will be removed in version 7.0.0.", true)]
         public bool HasNativePubSubSupport { get; set; }
-        [System.ObsoleteAttribute("Please use `GetOutboundRoutingPolicy` instead. Will be removed in version 7.0.0.", true)]
+        [System.ObsoleteAttribute("This concept is no longer in use, you can replace it with TransportInfrastructure" +
```

Comment:
How about "the concept of CentralizedPubSub support is no longer available, please contact us ...." ?